src/domain/orders/details/claim/create.tsx

```diff
@@ -355,6 +353,7 @@ const ClaimMenu: React.FC<ClaimMenuProps> = ({ order, onDismiss }) => {
             />
             {returnShippingMethod && (
               <RMAShippingPrice
+                inclTax={false}
```

Comment:
question: If we default to tax-exclusive, should we add a note/tooltip on it?

Reply:
Re flag, since it hasn't been there previously, let's keep that for a follow-up iteration.  

Re modal issues, I think that makes sense. Let's not make too many duct tape solutions now, if we have a larger revamp coming soon.

So this PR will be isolated to only fixing the null on claim items right?